tensorflow/lite/experimental/examples/lstm/rnn_cell.py

```diff
@@ -439,7 +439,7 @@ def call(self, inputs, state):
         index_override=18)
 
     input_size = inputs.shape.with_rank(2)[1]
-    if input_size.value is None:
+    if not isinstance(input_size, int) and input_size.value is None:
```

Comment:
thanks for the commit, how about we do:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/rnn_cell_impl.py#L1022-L1024

Reply:
Yeah, that works fine. Should I change it in my patch or are you going to use that code instead?

Reply:
can you change that? thanks a lot!

Reply:
Done.